ci-operator/config/openshift/installer/openshift-installer-release-4.14.yaml

```diff
@@ -486,6 +486,32 @@ tests:
     cluster_profile: openstack-vh-mecha-central
     workflow: openshift-e2e-openstack-proxy
   timeout: 6h0m0s
+- always_run: false
+  as: e2e-openstack-ovn-masterscale
+  optional: true
+  steps:
+    cluster_profile: openstack-vexxhost
+    env:
+      BASE_DOMAIN: shiftstack.devcluster.openshift.com
+      CONFIG_TYPE: minimal
+      ROTATE_CLOUD_CREDENTIALS: "true"
+      SCALE_MASTERS_FLAVOR_ALTERNATE: "true"
+      TEST_SUITE: experimental/reliability/minimal
+    workflow: openshift-e2e-openstack-ipi
+  timeout: 6h0m0s
+- always_run: false
+  as: e2e-openstack-ovn-masterscale-zones
```

Comment:
I don't recall what was exactly your intent behind `e2e-openstack-ovn-masterscale-zones` exactly. If you can help me clarify so I can finish this PR. Thanks

Reply:
oh I read it in the commit message, I just changed the cluster profile to use a cloud which has a zone.